forms.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Page Title</title>
+<meta charset="UTF-8">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="stylesheet" href="mystyle.css">
+</head>
+<body>
+
+<div class="header">
+  <h1>My Website</h1>
+  <img src="./images/adoptadog-logo.svg" alt="Logo" width="1500" height="800">
+  <a href="./forms.html" class="button">Checkout</a>
+  <p>A website created by me.</p>
+</div>
+
+<div class="navbar">
+  <a href="index.html">Home</a>
+  <a href="./dogs.html">Dogs</a>
+  <a href="./blogs.html">Blog</a>
+</div>
+<h3>Checkout</h3>
+
+<div class="container">
+  <form action="/action_page.php">
```

Comment:
this php might cause you trouble down the line. It's fine for now, but in a month or so we will deal with some form stuff and having PHP MIGHT fight with your JS. Just a heads up.